.clippy.toml

```diff
@@ -0,0 +1,8 @@
+# NOTE: Other global Clippy config is in top-level Cargo.toml.
+
+disallowed-types = [
+    { path = "std::collections::HashMap", reason = "use hashbrown::HashMap instead" },
+    { path = "std::collections::HashSet", reason = "use hashbrown::HashSet instead" },
+    { path = "rustc_hash::FxHashMap", reason = "use hashbrown::HashMap instead" },
+    { path = "rustc_hash::FxHashSet", reason = "use hashbrown::HashSet instead" },
+]
```

Comment:
I am inclined to rename this to `clippy.toml` with no leading dot, as I said in the other comment.

Reply:
Does clippy care? If not, then no dot is fine.